mcp/src/minecraft/com/elementfx/tvp/ad/client/gui/inventory/GuiRohirrimCrafting.java

```diff
@@ -0,0 +1,49 @@
+package com.elementfx.tvp.ad.client.gui.inventory;
+
+import com.elementfx.tvp.ad.inventory.ContainerElvenWorkbench;
+import com.elementfx.tvp.ad.inventory.ContainerRohirrimWorkbench;
+
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.inventory.ContainerWorkbench;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+public class GuiRohirrimCrafting extends GuiContainer
+{
+	private static final ResourceLocation ROHIRRIM_CRAFTING_TABLE_GUI_TEXTURES = new ResourceLocation("textures/gui/container/crafting_table.png");
```

Comment:
Do you think that there will be any differences between the GuiXCrafting aside from the background texture and font color? If we're not going to customize them any more than that, they we can probably get away with a single class that passes those two things into the constructor.